src/test/java/com/google/cloud/teleport/spanner/ExportRelatedTablesCheckTest.java

```diff
@@ -124,38 +124,105 @@ private void createAndPopulate(Ddl ddl, int numBatches) throws Exception {
   /* Validates behavior of exporting full db without selecting any tables */
```

Comment:
Are you able to revert this file? I'm guessing you ran spotless (thanks!), but we generally prefer the old way of formatting. I must have missed guarding these tests against spotless formatting.